source/_posts/2017-05-19-kanban-method-in-a-consulting-company.markdown

```diff
@@ -0,0 +1,100 @@
+---
+author: Michał Olszewski
+layout: post
+title: "Kanban method in a consulting company"
+date: 2017-05-12 12:00:00 +0100
+comments: true
+categories: Managament 
+---
+
+# Kanban method in a consulting company
+
+
+
+Before we start we need to know what is that Kanban. Let me show you a short 
+explanation:
+
+> **Kanban** is a method for managing resources necessary for production process
+so that everyone produces much as need at a given moment. Work items are 
+visualized to give participants a view of progress and process, from task 
+definition to customer delivery. Team members "pull" work as capacity permits, 
+rather than work being "pushed" into the process when requested.
+
+> In software development, Kanban provides a visual process-management system
+which aids decision-making about what, when and how much to produce. Although
+the method (inspired by the Toyota Production System and lean manufacturing
+originated in software development and IT, it may be applied to any professional 
+service whose work outcome is intangible rather than physical.
+
+## Tools
+
+The basic work tools are boards and stickers, but in our company we use visual 
+boards in [Trello] (https://trello.com/), which streamlines our work in kanban. 
+Our arrays are divided into appropriate categories: Backlog, To Do, In Progress,
+Verification, Done and Project Status, where we can discuss project features 
+or problems.
+
+![1](http://imageshack.com/a/img922/6156/YfxSE1.png)
+
+## Cards (Kanbans)
+
+According to the Kanban method we give out a kanban cards to assigned 
+employees. The card is very important in the whole project process. According
+to her, we measure our work and set time for each stage in the design, develop
+and testing solutions in embedded systems projects. Additionally inside the 
+card we can discuss the features and see, what happened since last time.
+
+![img](http://imageshack.com/a/img924/2586/spqDrv.png)
+
+## Workflow
+
+The card life begins in ToDo, sometimes it happens there we have a lot of big
+task and cards with small priority are moved to the Backlog, where they are 
+waiting for some free time to solve them. We are trying to minimize transfering
+the cards from right to left, our goal is to streamline this process.
+
+![kanban-flow](http://imageshack.com/a/img923/2471/u20Zzr.jpg)
+
+## Rules
+
+To make kanban work, each employee must follow these rules:
+
+- [ ] Do not take a lot of tasks,
+- [ ] Keep order,
+- [ ] Discuss about tasks,
+- [ ] Don't race for who will do more - focus on teamwork,
+- [ ] When somebody have problem our task is to help each other.
+
+These rules are only a sample but improve the whole process. We know situations 
+where tasks is going fast to verification, after which it turns out that the 
+task is incomplete and we need turn it back to the left. Good comunication 
+helps, what if some feature is expired and we dont know about it? One sentence 
+in communication can save much of time. 
```

Comment:
@molszewski9412 concept of this paragraph is good but should be rephrased, because it sounds very lame. Please describe specific case and explain how applying rules we are able to do better job.